src/lib/slide-toggle/slide-toggle.ts

```diff
@@ -125,6 +125,7 @@ export class MdSlideToggle extends _MdSlideToggleMixinBase implements OnDestroy,
   @Input()
   get checked(): boolean { return this._checked; }
   set checked(value) {
+    value = coerceBooleanProperty(value);
     this._checked = !!value;
```

Comment:
`this._checked = coerceBooleanProperty(value);`?

Reply:
Yep, it should be that

Reply:
Done :)

Reply:
`coerceBooleanProperty` already returns a boolean, you don't need `!!` :)

Reply:
#facepalm